### PR TITLE
001 achievments view

### PR DESCRIPTION
**Pull request by HugeErick**
# Waiting for IA

request for pulling current Achievements view

### CODE REVIEW

**Comment by DaWaildHorse** [comment]
please make the following changes: 

> Use the correct palette in AchievementView.swift

File: achievments.txt
Comment (DaWaildHorse) [inline achievments.txt:1]: Please use the same verbs 

> Instead of Capture use Take a ...

**Review by DaWaildHorse — CHANGES_REQUESTED** [review]

File: achievments.txt
Comment (HugeErick) [reply achievments.txt:1]: ?????????????????????????????????????????????????????????????????????????????????????????????????????????????????????

Comment (HugeErick) [reply achievments.txt:1]: no

Comment (DaWaildHorse) [reply achievments.txt:1]: please check the achievments.txt. there your will see mi comments

Comment (DaWaildHorse) [reply achievments.txt:1]: u are useless, i will do it myself

**Review by DaWaildHorse — APPROVED** [review]

**Comment by DaWaildHorse** [comment]
Good work, soon you wont be so bad as a swift dev. Please use correct wirting and write commments. Good work 

(This was all joke , i just wanted to test this haha)

**Pull request merged**